plugin.video.srgssr_ch_replay/resources/lib/router.py

```diff
@@ -0,0 +1,82 @@
+"""Routing in the plugin menus"""
+
+import os.path
```

Comment:
import os.path not used

Reply:
and lib/__init__.py is empty...

Reply:
if i'm not mistaking, the `__init__.py` is needed to make a python package from a directory and then import `from resources.lib.plugin import Plugin` in the `addon.py` file.